src/main/kotlin/no/nav/syfo/cronjob/DialogmoteCronjobRunner.kt

```diff
@@ -26,7 +25,7 @@ class DialogmoteCronjobRunner(
         delay(initialDelay)
 
         while (applicationState.ready) {
-            val job = launch { run(cronjob) }
+            val job = launch(Dispatchers.Unbounded) { run(cronjob) }
```

Comment:
Antar at det er lurt å bruke den nye dispatcher'en her også.

Reply:
Tror du det er nødvendig? DialogmoteCronjobRunner#start kjøres jo fra en backgroundtask som er startet med en slik dispatcher, og så ser det ut som denne bruker coroutineScope derfra. Men jeg er ikke sikker 🤷‍♂️

Reply:
Godt poeng, kan revertere denne delen - så får vi eventuelt vurdere det på nytt dersom vi erfarer at cronjobber ikke oppfører seg som forventet. Dokumentasjonen sier: "If the context does not have any dispatcher nor any other ContinuationInterceptor, then Dispatchers.Default is used."

Reply:
Ja, sikkert greit å se an hvordan det oppfører seg 👍